src/pages/GenerateKeys/index.tsx

```diff
@@ -93,6 +130,22 @@ const _GenerateKeysPage = ({
     setMnemonicAcknowledgementChecked(e.target.checked);
   };
 
+  const handleAddressChange = (e: any) => {
+    const value = e.target.value.replace(/[^0-9a-fx]/gi, '');
```

Comment:
Curious about this regex, couldn't understand the purpose of it. Perhaps a comment could help.

Reply:
@pettinarip Basically a hexadecimal filter, also including `x` for the `0x` prefix
- `[]` declares a set of values I'm looking for
- `[^]` opening with a caret negates the set, so looking for any characters that don't match
- `0-9` any number, `a-f` letters "a" through "f", `x` also allow "x"
- `gi` ignore case and search the entire string
- `replace(/[^0-9a-fx]/gi, '')` Take any character that isn't found in an ETH address and remove it

Reply:
I can add a comment here so it helps clear this up for folks in the future

Reply:
Oh now I got it. Thanks for the explanation.